### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by Tectu** [stlink-org/stlink#1345] [Q&A (resolved)] [answered]
Is there a roadmap for the next release? Would be awesome to get distro packages updated.

**Comment by Nightwalker-87** [comment]
No, I'm afraid there isn't for the time being. There are not enough contributors around to resolve the remaining issues, mainly bugfixes, targeted for such a release. This makes it very difficult to generate a roadmap including the time frame necessary for testing.

**Comment by Nightwalker-87** [comment]
@Tectu: With recent progress in maintenance work, it has now become very likely that there will be a new release somewhen in Q1/2024. (I know it's somehow overdue.) However, I'd like to resolve two remaining issues first, before starting release preparations. Thus I can't point to any exact date yet... Please stay tuned.

**Comment by Nightwalker-87** [comment]
A new release has now been scheduled for rollout on 01.02.2024.

**Comment by Tectu** [comment]
That's great news! Thank you for all your efforts!

**Comment by Nightwalker-87** [comment]
@Tectu Release preparations are about to conclude.
The Windows build has been compiled and tested already. :relieved:
Looking at the .deb and .rpm packages now...

**Comment by Nightwalker-87** [comment, marked as answer]
The new release is now published. 🎉
Please mark this discussion as resolved.

**Comment by Tectu** [comment]
That's awesome - Thanks!
